addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -460,6 +460,24 @@
             </field>
         </record>
 
+        <!-- This view is specifically for sending mass mailings from different apps
+        like Contacts/CRM. And for that reason, priority is given high. -->
+        <record id="mailing_mailing_view_form_simplified" model="ir.ui.view">
```

Comment:
``mailing_mailing_view_form_nomodel`` maybe ? As more than simplified, it is used when model if "hardcoded" ?